api/v01/entities/vrp_input.rb

```diff
@@ -329,6 +329,16 @@ module VrpMissions
       use :vrp_request_quantity
     end
     mutually_exclusive :quantity_ids, :quantities
+
+    optional(:available_indices_interval,
```

Comment:
```suggestion
    optional(:range_indices,
```

Reply:
Otherwise, it is not uniform both with the configuration definition and the mutually_exclusive constraint

Reply:
I am not sure if these have the same meaning. 

scheduling configuration `range_indices` defines the horizon of the whole planning and these availability intervals are parts of the horizon where the service is "available" to be served. Is that the case @fonsecadeline ?

If that is the case, I would suggest the following naming `:available_index_range` and `:available_date_range`.